lib/services/hpcpackage_remain.pm

```diff
@@ -41,17 +41,8 @@ sub install_pkg {
     my $version = get_var('HDDVERSION');
     # Install all 'library wrappers' (packages matching -hpc, not having a version number with '_' after their name)
     my @pkginstall = split('\n', script_output q[zypper search -r SLE-Module-HPC] . $version . q[-Pool -r SLE-Module-HPC] . $version . q[-Updates | cut -d '|' -f 2 | sed -e 's/ *//g' | grep -E '.*-hpc.*' | grep -vE 'system|module|suse' |  grep -vE '.*_[[:digit:]]+_[[:digit:]]+.*gnu|.*_[[:digit:]]+_[[:digit:]]+.*-hpc' | grep -vE '.*-static$' | grep -vE '.*hpc-macros.*'], proceed_on_failure => 1, timeout => 180);
-    my $disable    = (script_run("snapper --help | grep disable-used-space")) ? '' : '--disable-used-space';
-    my @snaps      = split('\n', script_output("snapper list $disable | grep important=no |tail -n2 | awk \'{print \$1}\'"));
-    my $start_snap = $snaps[-1] + 1;
-    for (my $i = 1; $i < @pkginstall; $i = $i + 1) {
-        zypper_call("in $pkginstall[$i]");
-        # we need to delete the snapshot after zypper install or it will left
-        # lots of snapshot there.
-        @snaps = split('\n', script_output("snapper list $disable | grep important=no |tail -n2 | awk \'{print \$1}\'"));
-        my $end_snap = $snaps[-1];
-        script_run('snapper delete ' . $start_snap . '-' . $end_snap, timeout => 90) if ($end_snap > $start_snap);
-    }
+    @pkginstall = grep { !/LMOD_SH_DBG_ON/ } @pkginstall;
```

Comment:
If only in x86 console, please specify it and add comments to make it clear to others.

Reply:
added. thanks